rss2email/email.py

```diff
@@ -76,6 +79,34 @@ def guess_encoding(string, encodings=('US-ASCII', 'UTF-8')):
             return encoding
     raise _error.NoValidEncodingError(string=string, encodings=encodings)
 
+def _add_plain_multipart(guid: str, message, html: str):
+    headers = message.items()
+    msg = MIMEMultipart('related')
+    for name, value in headers:
+        if name.lower().startswith('content-'):
+            continue
+        msg[str(name)] = value
+
+    alternative = MIMEMultipart('alternative')
+    msg.attach(alternative)
+
+    html_part = _MIMEText(html, _subtype='html')
+    alternative.attach(html_part)
+
+    text_content = html2text.html2text(html=html, baseurl=guid)
+    text_part = _MIMEText(text_content)
+    alternative.attach(text_part)
+    return msg
+
+def message_add_plain_multipart(guid, message, html):
+    if message.get_content_type() == 'text/html':
+        m = _add_plain_multipart(guid, message, html)
+        return m
+    if message.is_multipart():
+        # we could support multipart messages, but let's postpone it
```

Comment:
Maybe log a warning message?

Reply:
sounds like the right thing to do. In fact, I don't expect any message arriving in that part of the code to be a multipart message.